Libs/Widgets/Resources/ctkWidgets.qrc

```diff
@@ -5,8 +5,6 @@
         <file>Icons/unlock.png</file>
         <file>Icons/minus.png</file>
         <file>Icons/plus.png</file>
-        <file>Icons/expand-down.png</file>
-        <file>Icons/expand-up.png</file>
```

Comment:
Consider adding a note in the commit message explaining why these can be removed.

Reply:
Considering that local build of Slicer against this branch was :heavy_check_mark:, I will update the commit message like this and move forward with the integration:

```diff
- expand-down.png and expand-up.png are being removed as they were only used in ctkCollapsibleGroupBoxes built with Qt < 4.6
+ expand-down.png and expand-up.png are being removed as they were only used in ctkCollapsibleGroupBox built with Qt < 4.6
```